### ISSUE TITLE
has/whereHas error 'Array to string conversion' when used with relation belongsTo

### ISSUE THREAD

**Issue by chijovka** [topclaudy/compoships#57]
Hello
I need to filter the objects by relation belongsTo.
For example, you can take 2 models from your phpunit test (PickupTime and PickupPoint).
Models are left as is. In ComposhipsTest.php add method
```
public function testHasWithBelongsToRelation()
{
  $pickup_times = PickupTime::has('pickupPoint')->get()->toArray();

  $this->assertInternalType('array', $pickup_times);
}
```
As a result, the test falls with an error 

>  ErrorException: Array to string conversion

I use Laravel 5.4 and compoships 1.1.12

How could I do this work?

**Comment by topclaudy** [comment]
Hello @chijovka,

Thanks for reporting this. The issue is related to Laravel 5.4. This is fixed in 1.1.13.

**Comment by chijovka** [comment]
@topclaudy  Thanks. Everything is working
